WikiWatershed/model-my-watershed
# Compare View Revamp: TR-55 Runoff Charts and Tables

## Overview

Updated version of #2123. For more granular albeit noisy commits, see here: https://github.com/WikiWatershed/model-my-watershed/compare/develop...tt+ki/compare-view-tr55-runoff

Connects #2072 

## Testing Instructions

 * Check out this branch
 * Open a saved project, or create a new one, using TR-55 as the model and at least 2 scenarios
 * Open the Compare View
 * Ensure the values are correct in both the chart and the table
 * Move the precipitation slider, ensure the values update correctly
 * Ensure that the values of the original scenarios do not change after closing the Compare View
